src/frontend/src/lib/api/idb.api.ts

```diff
@@ -1,13 +1,15 @@
 import { browser } from '$app/environment';
+import { IDB_ADDRESS_STORE, IDB_ETH_ADDRESS_STORE } from '$lib/constants/idb.constants';
 import type { IdbEthAddress } from '$lib/types/idb';
 import type { Principal } from '@dfinity/principal';
 import { isNullish } from '@dfinity/utils';
 import { createStore, del, get, set, update, type UseStore } from 'idb-keyval';
 
 // There is no IndexedDB in SSG. Since this initialization occurs at the module's root, SvelteKit would encounter an error during the dapp bundling process, specifically a "ReferenceError [Error]: indexedDB is not defined". Therefore, the object for bundling on NodeJS side.
-const oisyEthAddressesStore = browser
-	? createStore('oisy-eth-addresses', 'eth-addresses')
-	: ({} as unknown as UseStore);
+const oisyAddressesStore = (storeName: string) =>
```

Comment:
I would add the term `Idb` in that function name to makes its purpose more obvious

Reply:
done